source/speech.py

```diff
@@ -82,29 +83,25 @@ def processText(locale,text,symbolLevel):
 	return text.strip()
 
 def getLastSpeechIndex():
-	"""Gets the last index passed by the synthesizer. Indexing is used so that its possible to find out when a certain peace of text has been spoken yet. Usually the character position of the text is passed to speak functions as the index.
-@returns: the last index encountered
-@rtype: int
-"""
-	return getSynth().lastIndex
+	"""@deprecated: Use L{CallbackCommand} (or one of the other subclasses of L{BaseCallbackCommand}) instead.

```

Comment:
Could this be considered a left-over of speechCompat? As the API changes in a major way, you may as well remove this I guess.